src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -445,60 +445,104 @@ private unsafe int HandleEventConnected(ref CONNECTED_DATA data)
         QuicAddr localAddress = MsQuicHelpers.GetMsQuicParameter<QuicAddr>(_handle, QUIC_PARAM_CONN_LOCAL_ADDRESS);
         _localEndPoint = localAddress.ToIPEndPoint();
 
-        _connectedTcs.TrySetResult();
-
         if (NetEventSource.Log.IsEnabled())
         {
-            NetEventSource.Info(this, $"{this} Connection connected {LocalEndPoint} -> {RemoteEndPoint}");
+            NetEventSource.Error(this, $"{this} Received event CONNECTED {LocalEndPoint} -> {RemoteEndPoint}");
         }
+
+        _connectedTcs.TrySetResult();
         return QUIC_STATUS_SUCCESS;
     }
+
```

Comment:
If you insist on putting empty line in between every HandleXYZ then it should be done everywhere. I was intentional with this, I don't mind the change but it should be the same everywhere.